koku/masu/processor/azure/azure_report_processor.py

```diff
@@ -314,6 +320,16 @@ def process(self):
                 for row in reader:
                     if not self._should_process_row(row, "UsageDateTime", is_full_month):
                         continue
+                    li_usage_dt = row.get("UsageDateTime")
+                    if li_usage_dt:
+                        try:
+                            li_usage_dt = parser.parse(li_usage_dt).date().replace(day=1)
```

Comment:
Can you switch this to using ciso8601? Datetime handling in Python it turns out is quite slow. Dropping down to C with that library has actually helped speed up some bottlenecks in report processing. Should be an example of it in this file.

Reply:
OK, thanks!!